src/mds/objects.py

```diff
@@ -221,6 +330,60 @@ async def _create_aliases_for_record(
     logger.info(f"added aliases: {aliases} for guid: {blank_guid}")
 
 
+async def _create_blank_version(
+    indexd_record: dict,
+    file_name: str,
+    uploader: str,
+    auth_header: str,
+    request: Request,
+):
+    """
+    Create a new, blank version of the provided indexd object.
+    """
+    guid = indexd_record["did"]
+    logger.debug(f"trying to create a new blank record version for {guid}")
+    try:
+        endpoint = config.INDEXING_SERVICE_ENDPOINT.rstrip("/") + f"/index/blank/{guid}"
+        data = {
+            "uploader": uploader,
+            "file_name": file_name,
+            # NOTE: cannot set ACL of blank versions yet
```

Comment:
I don't know that we ever want to support that if we're pushing towards using authz everywhere. I'm inclined to just say leave this out entirely or word it like "we won't support setting deprecated acl field"